Python/samples/quanser-qube/selector.ink

```diff
@@ -0,0 +1,170 @@
+###
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+
+# Multi-concept with programmed selector architecture using Quanser Qube.
+# Train the SwingUp Concept
+# Train the Balance Concept
+# Build a Programmed Concept for selecting between the two
+
+###
+
+inkling "2.0"
+using Number
+using Math
+using Goal
+
+# Constant threshold for desired region of pendulum, where 0 is vertical
+const alpha_balance_threshold = 12 
+
+# Constant threshold for defining terminal condition of motor
+const theta_rotation_threshold = 90
+
+# State from the sim and for the brain
+type ObservableState {
+    theta: number, # radians, motor angle
+    alpha: number, # radians, pendulum angle
+    theta_dot: number, # radians / s, motor angular velocity
+    alpha_dot: number, # radians / s, pendulum angular velocity
+}
+
+# Action type definition is the same for both concepts
+type BrainAction {
+    Vm: Number.Float32<-3 .. 3> # Voltage of motor
+}
+
+# Simulator configuration 
+type SimConfig {
+    Lp: number, # m, length of pole
+    mp: number, #kg, mass of pole
+    Rm: number, # Resistance
+    kt: number, # Current-torque (N-m/A)
+    km: number, # Back-emf constant (V-s/rad)
+    mr: number, # Mass (kg)
+    Lr: number, # Total length (m)
+    Dr: number, # Equivalent viscous damping coefficient (N-m-s/rad)
+    Dp: number, # Equivalent viscous damping coefficient (N-m-s/rad)
+    frequency: number, # Hertz
+    initial_theta: number, # radians, motor angle
+    initial_alpha: number, # radians, pendulum angle. 0 : vertical, pi : down
+    initial_theta_dot: number, # radians / s, motor angular velocity
+    initial_alpha_dot: number, # radians / s, pendulum angular velocity
+}
+
+# Function to convert Degrees to Radians for constants given in Degrees 
+function DegreesToRadians (Degrees: number): number {
+    return Degrees * Math.Pi / 180
+}
+
+# Function for Linear Quadratic Regulator (Programmed Controller not used)
+function LQR(State: ObservableState): BrainAction {
```

Comment:
Can this be used as a controller instead of the trained brains + the selector? Or is it just listed here for reference?

Reply:
@akzaidi This can be used as a controller. But it is not used in this demo at the moment.